packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -816,7 +816,13 @@ export function TransferPanelMain({
             )}
             onChange={networkListboxProps.from.onChange}
           >
-            <span className="max-w-[220px] truncate md:max-w-[250px]">
+            <span
+              className={twMerge(
+                'max-w-[220px] truncate md:max-w-[250px]',
+                // temporary fix not to truncate their network name, this will be resolved by the reskin
+                networks.sourceChain.id === 70700 && 'text-[20px]'
```

Comment:
Wouldn't it be better to have
`getNetworkName(networks.sourceChain.id).length > 15  && 'text-[20px]'`

Reply:
Discussed offline, will target 70700 only. The reskin will remove the need for this anyway.